custom/opm/reports.py

```diff
@@ -961,6 +963,38 @@ def format_cell(self, val, denom):
         pct = " ({:.0%})".format(float(val) / denom) if denom != 0 else ""
         return "{} / {}{}".format(val, denom, pct)
 
+    @property
+    def export_table(self):
+        """
+        Split the cells up into 3 for excel
+        """
```

Comment:
if there's no denominator specified in the spec, just make it show up as one column for that indicator.  Otherwise do three columns - value, denominator, percent.